apps/filemanager/templates/deployment.yaml

```diff
@@ -20,6 +20,7 @@ spec:
     metadata:
       labels:
         release: {{ .Release.Name }}
+        access: {{ .Values.permission }}
```

Comment:
I don't think this is needed for the filemanager
<img width="983" alt="Screenshot 2024-09-18 at 13 36 11" src="https://github.com/user-attachments/assets/88727662-f392-4c60-99ca-33f1e2dbc332">